src/Evolution/Systems/NewtonianEuler/Initialize.hpp

```diff
@@ -0,0 +1,166 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <tuple>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Mesh.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/Initialization/ConservativeSystem.hpp"
+#include "Evolution/Initialization/DiscontinuousGalerkin.hpp"
+#include "Evolution/Initialization/Domain.hpp"
+#include "Evolution/Initialization/Evolution.hpp"
+#include "Evolution/Initialization/InterfaceNewtonianEuler.hpp"
+#include "Evolution/Initialization/Limiter.hpp"
+#include "Evolution/Systems/NewtonianEuler/SoundSpeedSquared.hpp"
+#include "Evolution/Systems/NewtonianEuler/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/Divergence.tpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "PointwiseFunctions/AnalyticData/Tags.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace NewtonianEuler {
+namespace Actions {
+namespace detail {
+template <class T, class = cpp17::void_t<>>
+struct has_analytic_solution_alias : std::false_type {};
+template <class T>
+struct has_analytic_solution_alias<T,
+                                   cpp17::void_t<typename T::analytic_solution>>
+    : std::true_type {};
+}  // namespace detail
+
+template <size_t Dim>
+struct Initialize {
```

Comment:
Can you please file an issue to remove this once PDALs are in? This code is almost identical to the GRMHD case and I think we should figure out how to not do that once we have PDALs :)

Reply:
Sure, I'll also mention that it should be removed from GRMHD (and possibly from GH, if it gets merged soon).

Reply:
Thanks!